superset/views/base.py

```diff
@@ -312,6 +312,11 @@ def menu_data() -> Dict[str, Any]:
     brand_text = appbuilder.app.config["LOGO_RIGHT_TEXT"]
     if callable(brand_text):
         brand_text = brand_text()
+    build_number = ""
+    try:
+        build_number = appbuilder.app.config["BUILD_NUMBER"]
+    except Exception as ex:
+        logger.debug("BUILD_NUMBER is missing from the config", ex)
```

Comment:
this shouldn't be necessary if you add a default value in config.py, (either empty string or `None`). Config values should never be undefined.

Reply:
Done.

Reply:
Anything required at our end at this point?

Reply:
Yes, you need to get CI passing before we can merge